### PR TITLE
Added typings for AccountApi 

### LINKED ISSUE #359: Account Types
Create typings for Account API.

### PR DESCRIPTION
**Pull request by bpedersen**
Added typings to AccountAPI

## Description
Added typings for the AccountAPI from what I could find online

## Motivation and Context
Hacktoberfest (https://developer.nexmo.com/hacktoberfest)
This was requested by https://github.com/Vonage/vonage-node-sdk/issues/359

## Testing Details
No tests should be affected

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by kellyjandrews** [comment]
Thanks for the contribution - we will get it reviewed and merged as soon as we can!

**Comment by bpedersen** [comment]
Thank you for the Hacktoberfest tag, however it looks like they changed the requirements recently. 
Looks like we need to use a tag for PRs as Hacktoberfest-Accepted
https://hacktoberfest.digitalocean.com/hacktoberfest-update

**Comment by mheap** [comment]
Good news! The `vonage-node-sdk` repo has the `hacktoberfest` topic, so all PRs submitted against this repo are implicitly eligible to be counted

File: typings/index.d.ts [hunk 1]
```diff
@@ -11,205 +11,269 @@
...
+		createSecret(apiKey: string, secret: string, callback?: unknown): void;
+		deleteSecret(apiKey: string, id: string, callback?: unknown): void;
+	}
+
+
+	/* Vonage */
+	export default class Vonage {
+		constructor(credentials: CredentialsObject, options?: { [key: string]: any });
+		public readonly verify: Verify;
+		public readonly message: Message;
```
Comment (kellyjandrews) [inline typings/index.d.ts:277]: You will also need to add the account type to the default export here.

**Review by kellyjandrews — CHANGES_REQUESTED** [review]
Looks like this is pretty close - just add the account module to the Vonage export and it should be good to go.

**Comment by garann** [comment]
> Thank you for the Hacktoberfest tag, however it looks like they changed the requirements recently.
> Looks like we need to use a tag for PRs as Hacktoberfest-Accepted
> https://hacktoberfest.digitalocean.com/hacktoberfest-update

Hi @bpedersen, just checking on this PR and https://github.com/Vonage/vonage-node-sdk/pull/374. I see both have changes requested, and once those are made it should be possible to get them accepted and merged. Are you still hoping to get these accepted before the end of the month? Thank you!

**Pull request closed without merging**